dashboard/stack.yml

```diff
@@ -38,7 +51,4 @@ functions:
     environment:
       content_type: "text/html"
       combine_output: "false"
-      gateway_url: http://gateway.openfaas:8080/
-
-# For Swarm use URL below which does not include .openfaas suffix:
```

Comment:
Please insert this comment again if it's missing in the new version

Reply:
Looks like I missed reverting that part. Fixed.